src/test/java/com/google/sps/ManageAvailabilityTest.java

```diff
@@ -120,7 +119,6 @@ public void testDoPost() throws Exception {
         servlet.doPost(request, response);
 
         verify(request, times(1)).getParameter("startTime");
-        verify(request, times(1)).getParameter("endTime");
         verify(request, times(1)).getParameter("date");
 
         Calendar expectedDate = new Calendar.Builder()
```

Comment:
I might have missed it, but are we adding the checks to make sure the tutor doesn't add an availability in the past?